### DISCUSSION TITLE
Missing lib files during initial 'forge init'.

### DISCUSSION THREAD

**Discussion by sepia** [foundry-rs/foundry#8663] [Help] [answered]
Hey peoples. I'm fairly new in using foundry. Whenever I try to create a foundry project(forge init) the lib files are not downloaded. All I get is the lib 'folder' but it's empty. Can someone here explain what I'm doing wrong. Thanks in advance.

**Comment by urosognjenovic** [comment]
Try running ```forge update forge-std``` to update the ```forge-std``` library.

If this doesn't help, try running ```foundryup``` to install the latest versions of ```forge```, ```cast```, ```anvil```, and ```chisel```.

**Reply by sepia** [reply]
Thanks, thebraveman17. I'll give that a try and let you know the outcome.

**Reply by sepia** [reply]
I tried 'forge update forge-std in vs code terminal and got the following error, "Error: 
git rev-parse exited with code 128:
fatal: not a git repository (or any of the parent directories): .git"
Tried foundryup and got the following error, "foundryup : The term 'foundryup' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the 
name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ foundryup
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (foundryup:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException"

Again, what am I doing wrong? Thanks.

**Reply by zerosnacks** [reply, marked as answer]
Hi @sepia, it looks like you have a wrong version of Foundry installed.

Please follow the installation instructions here: https://book.getfoundry.sh/getting-started/installation#using-foundryup and please read the note in regards to using Foundry on Windows
